cmd/apiserver/app/server.go

```diff
@@ -0,0 +1,143 @@
+/*
+Copyright 2022 The KubeVela Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package app
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"os"
+	"os/signal"
+	"syscall"
+
+	"sigs.k8s.io/controller-runtime/pkg/manager/signals"
+
+	restfulspec "github.com/emicklei/go-restful-openapi/v2"
+	"github.com/fatih/color"
+	"github.com/go-openapi/spec"
+	"github.com/spf13/cobra"
+
+	"github.com/oam-dev/kubevela/cmd/apiserver/app/options"
+	"github.com/oam-dev/kubevela/pkg/apiserver"
+	"github.com/oam-dev/kubevela/pkg/apiserver/utils/log"
+	"github.com/oam-dev/kubevela/pkg/utils"
+	"github.com/oam-dev/kubevela/version"
+)
+
+// NewAPIServerCommand creates a *cobra.Command object with default parameters
+func NewAPIServerCommand() *cobra.Command {
+	s := options.NewServerRunOptions()
+	cmd := &cobra.Command{
+		Use: "apiserver",
+		Long: `The KubeVela API server validates and configures data for the API objects. 
+The API Server services REST operations and provides the frontend to the
+cluster's shared state through which all other components interact.`,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			if len(args) > 1 && args[0] == "build-swagger" {
```

Comment:
Can we make the "build-swagger" a standalone command?

Reply:
Of course, it has been done